packages/plugin/src/theia.d.ts

```diff
@@ -5323,6 +5323,19 @@ export module '@theia/plugin' {
          */
         password: boolean;
 
+
```

Comment:
Minor: Our linter complains here that there is more than one consecutive empty line here (below `valueSelection` as well).

Reply:
removed the lines